data/adversaries/7e422753-ad7a-4401-bc8b-b12a28e69c25.yml

```diff
@@ -1,8 +1,8 @@
 id: 7e422753-ad7a-4401-bc8b-b12a28e69c25
-name: Incident responder
-description: A basic incident responder profile
-phases:
-  1:
-    - 3b4640bc-eacb-407a-a997-105e39788781
-  2:
-    - 02fb7fa9-8886-4330-9e65-fa7bb1bc5271
\ No newline at end of file
+        name: Incident responder
```

Comment:
looks like some odd spacing here. let's make it consistent

Reply:
my bad, didn't mean to commit this change